control_server/httpserver/routes.go

```diff
@@ -0,0 +1,142 @@
+package httpserver
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"syscall"
+
+	"github.com/gorilla/mux"
+
+	"baas/pkg/fs"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/google/uuid"
+
+	"baas/control_server/machines"
+	"baas/pkg/api"
+	"baas/pkg/model"
+)
+
+// Routes is a struct on which functions are defined that respond to requests
+// from the management OS. This struct holds state necessary for the request handlers.
+type Routes struct {
```

Comment:
Maybe a constructor would be nice as that would make it easier to add more advanced things in the future

Reply:
fixed